WorkflowTesting/Sources/WorkflowRenderTester.swift

```diff
@@ -170,6 +170,31 @@
             )
         }
 
+        /// Expect the given workflow type in the next rendering, with its output being ignored by a call to `ignoringOutput()`.
+        ///
+        /// - Parameters:
+        ///   - type: The type of the expected workflow.
+        ///   - key: The key of the expected workflow (if specified).
+        ///   - rendering: The rendering result that should be returned when the workflow of this type is rendered.
+        ///   - assertions: Additional assertions for the given workflow, if any. You may use this to assert the properties of the requested workflow are as expected.
+        public func expectWorkflowIgnoringOutput<ExpectedWorkflowType: Workflow>(
```

Comment:
@dhavalshreyas Is this in line with what you had in mind?

Reply:
Yes! This is it :)